regen-network/regen-web
# feat: add new web2 login tracking events

## Description

Closes: #2285


I've added all events belonging to the "web 2.0 log in" App Section (column D) from https://docs.google.com/spreadsheets/d/1JXw_pDC4QyqtcdYgxpMkbTVc24E2yz-qpwxvzelPfnY/edit#gid=465273655
That corresponds to rows 67 to 78.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2310--regen-marketplace.netlify.app/, based on https://docs.google.com/spreadsheets/d/1JXw_pDC4QyqtcdYgxpMkbTVc24E2yz-qpwxvzelPfnY/edit#gid=465273655, test that all events from "web 2.0 log in" App Section are emitted.

You need to check the [**development** app on google analytics](https://analytics.google.com/analytics/web/?authuser=1#/p340668480/realtime/overview?params=_u..nav%3Dmaui&collectionId=life-cycle). 
**Also make sure you don't have any tracker blocker enabled.**

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
